README.md

```diff
@@ -95,6 +95,10 @@ The system design for Farmhand's multiplayer functionality has been [detailed in
 
 Automation is done with [GitHub Actions](.github/workflows). All changes are tested and built upon Git push. Merges to `main` automatically deploy to Production (the `gh-pages` branch) upon a successful test run and build.
 
+### Smoke Testing
+
+All are welcome to help with smoke testing Farmhand to ensure the game is stable! All you need is to be logged into GitHub and open a [new Smoke Test Report issue](https://github.com/jeremyckahn/farmhand/issues/new?template=smoke-test-report.md). This issue will have interactive checkboxes that you can use to track your progress. Once smoke testing is complete, please close the issue.
```

Comment:
is any of the list critical/blocker for deploying? i was thinking of smoketesting as a pre-deploy thing, so maybe even just a line here about when/why you would do a smoketest would explain.

Reply:
Good thinking! I added a note about this in https://github.com/jeremyckahn/farmhand/commit/97c416e09cac4898336dcba29bfab89e0a702273.

> is any of the list critical/blocker for deploying?

I don't want to be too prescriptive with this process at the outset, as we don't even know if it will be used all that much. I think keeping it a bit looser will make folks more inclined to participate, and we'll be able to evaluate issues on a case-by-case basis. Let's improve this process as we go!